cancermuts/datasources.py

```diff
@@ -1209,9 +1216,19 @@ def __init__(self, version='2.1'):
 
         gnomADExomeAlleleFrequency.set_version_in_desc(self._version_str[version])
         gnomADGenomeAlleleFrequency.set_version_in_desc(self._version_str[version])
+        gnomADPopmaxExomeAlleleFrequency.set_version_in_desc(self._version_str[version])
+        gnomADPopmaxGenomeAlleleFrequency.set_version_in_desc(self._version_str[version])
 
     def add_metadata(self, sequence, md_type=['gnomad_exome_allele_frequency'], use_alias=None):
-        
+        if '2.1' in self._gnomad_version:
```

Comment:
I think we should rely on `self._version_metadata_compatability` to make this decision, instead of hardcoding checks on `2.1` or `3`, e.g.:

```
for md in md_type:
    if not md in self._version_metadata_compatability[self._gnomad_version]:
        raise ValueError(...)
```

if we keep things parametric we don't need to do anything else when we e.g. add new versions of gnomAD in the future (adding version 4 would just be a matter of adding some appropriate entries in the dictionaries above)